src/index.html

```diff
@@ -1,3 +1,4 @@
+<!-- #region head -->
 <!doctype html>
 <html lang="en">
   <head>
```

Comment:
The <!doctype html> declaration must be the very first thing in an HTML document, before the <html> tag.